server/index.js

```diff
@@ -10,6 +10,17 @@ app.listen(port, function() {
   console.log(`listening on port ${port}`);
 });
 
+app.get('/products/:id', (req, res) => {
+  var id = req.params.id;
+  db.fetch(id)
+    .then( product => {

```

Comment:
wow!! promises!! in server code!! no callbacks!! wait tbh this is cool i should do this. good job Jon